Pendulum/Views/Sheets/AddEventSheet.swift

```diff
@@ -393,6 +424,27 @@ struct AddEventSheet: View {
                     self.ignore = newValue.defaultIgnore
                 }
             }
+            .onChange(of: date) { _ in
```

Comment:
This makes me wish we could define an array of items to watch onChange :-p

Reply:
Me too 😅